tests/dotnet-sos/README.md

```diff
@@ -0,0 +1,52 @@
+### Debugging managed dotnet code
+
+## Prerequisites
+1. lldb:
+```
+sudo apt install lldb-8
+```
+2. dotnet-sos:
+```
+dotnet tool install -g dotnet-sos
+# Installs the SOS extension, and
+# updates .lldbinit file to load SOS on lldb startup
+dotnet-sos install
+```
+3. oelldb, SGX enclave extension for lldb:
+```
+git clone --branch working https://github.com/vtikoo/oelldb
+ln -s <path-to-oelldb-dir>/oelldb /usr/bin/oelldb
+```
+
+## Example
+
+```
+make
+make run-ext2-lldb
+```
+
+
+Dotnet runtime issues a lot of cpuid instructions, which generate a SIGILL inside the enclave. To handle these run after the mystikos process is launched -
+```
+pro hand -p true -s false -n false SIGILL
```

Comment:
Try to see if we can do this automatically as part of `oelldb` initialization.

It would be ideal if we can automatically ignore only the CPUID related SIGILL.

Reply:
`pro hand` way of handling SIGILL requires the debuggee process to exist, so can't do it oelldb init time.
Have added a python script `ignore_sigill.py` which does the equivalent and can be imported at init. However, it ignores SIGILL completely, couldn't find a way to make it selective.